alem_da_graduacao/servicos.tex

```diff
@@ -319,6 +319,22 @@ \subsubsection{Conta educacional no GitHub}
 Para conseguir isso basta utilizar seu e-mail acadêmico ao se inscrever pelo
 link \url{education.github.com}.
 
+\subsubsection{Google Apps for Education}
+
+Em 2016, a Unicamp firmou um acordo com a Google para disponibilizar o
+Google Apps for Education para estudantes, docentes e funcionários. Logo no
+começo das aulas, você vai receber uma conta, que pode acessar em
+\url{googleapps.unicamp.br}, com seu RA como login e a senha sendo a que você
+usa para entrar no sistema da DAC.
+
+Nessa conta, você vai ter acesso a tudo o que uma conta normal do Google tem,
+mas com \emph{armazenamento ilimitado} tanto no \textbf{Gmail} quanto no
+\textbf{Drive}! Ou seja, podemos armazenar tudo o que quisermos sem preocupação!
+Além disso, é mais um email acadêmico que você pode usar a bel-prazer.
+
+Tudo sobre a parceria, com detalhes, pode ser encontrado em 
+\url{sites.google.com/site/unicampgonnagafe/home}.
```

Comment:
pra resolver o latex quebrando a coluna aqui, você tem 3 opções:
- Tira o "com detalhes", que a linha vai quebrar logo depois do "em"
- Forçar uma quebra de linha depois do "em", com um `\\`
- Usar um bit.ly pra esse link (e pros outros que faltam, se der)

eu estou tendendo pra 3a opção, testei as outras aqui e o link ainda fica passando um pouco da coluna (menos do que está agora, mas ainda passa), mas é tua escolha.